spec/services/manifest_builder/by_resource_type/manifest_builder_scanned_maps_spec.rb

```diff
@@ -38,7 +38,7 @@
   context "when given a nested scanned map set" do
     subject(:manifest_builder) { described_class.new(query_service.find_by(id: scanned_map.id)) }
     let(:scanned_map) do
-      FactoryBot.create_for_repository(:scanned_map, description: "Test Description", member_ids: child.id)
+      FactoryBot.create_for_repository(:scanned_map, description: "Test Description", member_ids: child.id, start_canvas: child.id)
```

Comment:
It's not clear now why the test setup has the start canvas. Maybe update the context to say that it's mistakenly set as a start canvas? I don't know, it might be better to have a separate test so that it's clear this is one is sort of a happy path test, have a new one for this edge case.

Reply:
Good read, made a new one.